### PR TITLE
Add activity module

### PR DESCRIPTION
**Pull request by woomurf**
- add activity module
  - add activity api (not related event)
  - add nft record api

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -507,4 +512,12 @@ export type AppCreditInfo = {
...
\ No newline at end of file
+};
+
+export type ActivityNftInfo = {
+  chain: string;
+  network: string;
+  contractAddress: string;
+  tokenId: string;
+  userAddress: string;
+}
```
Comment (ehgmsdk20) [inline src/types.ts:523]: Can you add a new line at the end of the file?

File: src/activity.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      appId,
+      userId,
+      nftInfo,
+      label,
+      data,
+    }
+    const trailingUrl = 'nft/record';
+    return this.sendRequest(HttpMethod.POST, trailingUrl, body);
+  }
+}
```
Comment (ehgmsdk20) [inline src/activity.ts:40]: ditto

**Review by ehgmsdk20 — APPROVED** [review]
LGTM with minor comments, thanks!

File: src/types.ts [hunk 3]
```diff
@@ -507,4 +512,12 @@ export type AppCreditInfo = {
   burnedSupply: number,
   maxSupply: number | null,
   createdAt: number,
-};
\ No newline at end of file
+};
+
+export type ActivityNftInfo = {
+  chain: string;
+  network: string;
```
Comment (platfowner) [inline src/types.ts:519]: 👍

File: src/activity.ts [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      userId,
+      data,
+      activityType,
+      nftInfo,
+    };
+    const trailingUrl = '';
+    return this.sendRequest(HttpMethod.POST, trailingUrl, body);
+  }
+
+  addNftRecord(
```
Comment (platfowner) [inline src/activity.ts:23]: what about adding some comments about how add() and addNftRecord() use cases are different?

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks!

File: src/types.ts (hunk 1 above)
Comment (woomurf) [reply src/types.ts]: done. Thanks!

File: src/activity.ts (hunk 4 above)
Comment (woomurf) [reply src/activity.ts:39]: I added simple comments. Thanks!

**Comment by woomurf** [comment]
Thanks for the review!

**Pull request merged**